content/python/concepts/random-module/terms/uniform/uniform.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: '.uniform()'
+Description: 'Returns a pseudo-random floating-point number between two given numbers.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Random'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `.uniform()` method takes two numbers as arguments and returns a pseudo-random floating-point number between them. The result is inclusive of the first value, and possibly inclusive of the second value, depending on rounding.
+
+## Syntax
+
+```pseudo
+random.uniform(value1,value2)
+```
+
+Where `value1` and `value2` are numbers bounding the choice of a random floating-point number.
+
+## Example
+
```

Comment:
Can we add an explanation of the example? Doesn't have to be the one I'm suggesting.
```suggestion

In the example below, `.uniform()` is used to return a random floating-point number between 10 and 20:

```